client/src/reviews/ReviewsList.tsx

```diff
@@ -1,18 +1,40 @@
-import React from 'react';
+import React, { ReactElement, useContext } from 'react';
+import Contexts from '../contexts/Contexts';
+import { reviews } from '../models/reviews.interface';
 import ReviewSort from './ReviewSort';
 import ReviewTile from './ReviewTile';
 
-const ReviewsList: React.FC = () => (
-  <div className="reviewList">
-    <h2>Reviews List</h2>
-    <ReviewSort />
-    <div className="reviewTileContainer">
-      <ReviewTile />
-      <ReviewTile />
-    </div>
-    <button>Add Review</button>
-    <button>More Reviews</button>
-  </div>
-);
+const ReviewsList: React.FC<reviews> = () => {
+  // const [displayReviews, setDisplayReviews] = useState(false);
+  // const [numDisplayed, setNumDisplayed] = useState(2);
+  // const [filterType, setFilterType] = useState('relevant');
+
+
+  const { setModalContent } = useContext(Contexts.ModalContext);
+  const { reviews }: reviews | null = useContext(Contexts.ReviewsContext);
```

Comment:
Ideally,  I think the type annotation here should be moved to the context, just to make is more explicit that the context can provide a null value.  It seems a bit big for this, and maybe should be in a separate PR